<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>951</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QTextEdit" name="m_text_Ssh_Status">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>481</width>
     <height>301</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="m_btn_quit">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>440</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>exit</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_line_inputcmd">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>420</y>
     <width>361</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>511</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>ip:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="m_line_ip"/>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>usr:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="m_line_username"/>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>pwd:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="m_line_pwd"/>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>port:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="m_line_port"/>
    </item>
    <item>
     <widget class="QPushButton" name="m_btn_connect">
      <property name="text">
       <string>connect</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
